### ISSUE TITLE
[Bug]: Cannot connect to SMB server with android tv version

### ISSUE THREAD

**Issue by Pirokar** [nova-video-player/aos-AVP#689]
### Problem description

suddenly the application just stopped seeing the SMB server. VLC recognizes. The phone recognizes. Nova Video Player android tv version no.I've tried switching the SMB recognition order in the settings - it didn't help.

### Steps to reproduce the issue

Just try to watch some video from SMB server.

### Your phone/tablet/androidTV model

Nvidia shield 2019

### Operating system version

Android 9.1.0

### Application version and app store

6.0.71 from Google play for android tv

**Comment by courville** [comment]
Thanks for reporting. In order to understand the situation: do you confirm that the same version of nova on your phone sees the SMB share? Can you please disclose what SMB server you are using? Do you confirm that going on android tv in the network shares you have no server listed?

**Comment by Pirokar** [comment]
Yes, I can connect from nova phone version. I'm using Asus ax88u Merlin firmware with SMB server v1+v2.

**Comment by Pirokar** [comment]
Uninstalling and installing again helped me. But I think this bus still should be fixed :)

**Comment by Pirokar** [comment]
It has disappeared again :D

**Comment by Pentaphon** [comment]
Did you update to 6.0.97?
